docs/docs/12-context.zh-CN.md

```diff
@@ -55,16 +55,16 @@ var MessageList = React.createClass({
 });
 ```
 
-在这里例子里,我们手工穿透一个 `color` prop 以便于恰当格式化 `Button` 和 `Message` 组件.主题是一个很好的例子,当你可能想整个子树都可以访问一部分信息时(比如color). 使用 context 我们可以自动传过这个树:
+在这里例子里，我们手工传递一个 `color` prop， 以便于给 `Button` 和 `Message` 组件添加合适的样式 。当你想所有子组件可以访问一部分信息时(比如color)，设置主题是一个很好的例子。通过使用 context ，我们能自动在组件树中传递信息：
 
-```javascript{2-4,7,18,25-30,33}
+```javascript{2-4，7，18，25-30，33}
```

Comment:
We shouldn't use Chinese punctuation marks in code blocks. The comma above ("，") is a [fullwidth comma (U+FF0C)](https://en.wikipedia.org/wiki/Chinese_punctuation), and it probably won't be parsed correctly.